app/src/main/java/com/github/onlynotesswent/ui/common/NoteComposables.kt

```diff
@@ -74,25 +80,15 @@ fun NoteItem(
     navigationActions: NavigationActions,
     onClick: () -> Unit
 ) {
-  var showMoveOutDialog by remember { mutableStateOf(showDialog) }
+  var showBottomSheet by remember { mutableStateOf(false) }
 
-  if (showMoveOutDialog && note.folderId != null) {
-    ConfirmationPopup(
-        title = stringResource(R.string.move_note_out_of_folder),
-        text = stringResource(R.string.move_note_out_of_folder_confirmation),
-        onConfirm = {
-          val parentFolderId = folderViewModel.parentFolderId.value
-          if (parentFolderId != null) {
-            noteViewModel.updateNote(note.copy(folderId = parentFolderId))
-            navigationActions.navigateTo(
-                Screen.FOLDER_CONTENTS.replace(oldValue = "{folderId}", newValue = parentFolderId))
-          } else {
-            noteViewModel.updateNote(note.copy(folderId = null))
-            navigationActions.navigateTo(TopLevelDestinations.OVERVIEW)
-          }
-          showMoveOutDialog = false
-        },
-        onDismiss = { showMoveOutDialog = false })
+  if (showBottomSheet) {
+    NoteOptionsBottomSheet(
+        note = note,
+        noteViewModel = noteViewModel,
+        folderViewModel,
```

Comment:
for consistency, it is better to change this to `folderViewModel = folderViewModel` and the same for navigation actions

Reply:
Yes good point I changed it